README.md

```diff
@@ -42,11 +44,14 @@ Main config values:
     
 To push to github, you will need to configure SSH keys. See [Github's documentation}(https://help.github.com/articles/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent/)
 
+You may also provide values for these as environment variables named
+gitberg\_gh\_user etc.
+
 ### Development
 
 To run project in development mode clone the project and do:
-    
-    python setup.py develop
+
+    pip install .
```

Comment:
does that do the same thing?

Reply:
No. The setup.py one uses the dependency information there, vs the pip one uses requirements.pip. setup.py's dependency information is outdated (missing travispy for starters), and when I tried updating it I ran into issues as mentioned in the commit message.